### ISSUE TITLE
[bug] Hover image stays forever when deleting the image markdown

### ISSUE THREAD

**Issue by ndsvw** [CalebJohn/joplin-rich-markdown#69]
Hello, this feels a little bit unexpected:

---

Repro:

1) Add an image to your markdown document, so that it looks like this:

```
![Image_20240604_201316.webp](:/1b01f1bed2c64864ac295561191cedf7)
```

2) Hover over the markdown image representation, so that the image is shown to you
3) Double click on the markdown image representation
4) Press the delete key, so that the line for the whole image is removed

Expected:

- I cannot see the displayed image anymore

Actual:

- The displayed image stays there forever. It does not even go away when I switch to a different markdown note and come back.

**Comment by CalebJohn** [comment]
This will be fixed in the next release, I should have it out later today

**Comment by ndsvw** [comment]
Awesome!

Thanks, can confirm that it is fixed in 0.15.1
